src/locales/tr.yml

```
@@ -0,0 +1,524 @@
+rules: |-
+  ### 📜 Öğrenciler arası ağ kuralları 
+
+   1. ** Sağduyu kullanın: ** Başkalarını ve görüşlerini düşünün. Slurs, aşırı aşağılayıcı bir dil veya sohbetin rahatlığını bozabilecek herhangi bir eylem yok. 
+   2. ** Spam veya sel yok: ** Tekrarlanan, saçma veya aşırı uzun mesajlardan kaçının. 
+   3. ** Özel konuları özel tutun: ** Kişisel bilgileri ağ genelinde paylaşmaktan kaçının. 
+   4. ** Taciz Yok: ** Trolling, hakaret veya herhangi bir tür taciz tolere edilmez. 
+   5. ** NSFW/NSFL içeriği yok: ** Açık NSFW/NSFL içeriği yayınlamak anında kara listeye neden olacaktır. 
+   6. ** Hassas konulara saygı gösterin: ** Kendine zarar, intihar, şiddet veya diğer saldırı konularını önemsizleştirmeyin. 
+   7. ** Rapor Endişeleri: ** Bu kuralların ihlali gözlemliyorsanız, daha fazla eylem için uygun Hub Moderatörüne veya Motor Personeline bildirin. 
+
+   Sorusu olan? [Destek sunucumuz] ({{{support_invite}}}) katılın.
+vote: |-
+  ## 🗳️ İnterchat için oy verin ve özel avantajların tadını çıkarın 
+   Katılımınız, Interchat'in TOP.GG. Her oy önemli bir fark yaratıyor! 
+
+   Desteğiniz için minnettarlık ifade etme yolumuz olarak, size özel avantajlar sunmaktan heyecan duyuyoruz. İnterchat için oylamayı yayınlayarak kilidini açacaksınız: 
+
+   - Hub'lar içindeki mesajları düzenleyin 
+   - Çeviri mesajları (ve çok daha fazlası yolda!) 
+
+   Değişmez desteğinizi derinden takdir ediyoruz. Teşekkür ederim! 🙏
+network:
+  deleteSuccess: >-
+    {{{emoji}}}}} {{{user}}} tarafından mesaj __ ** {{deleted}}/{{total}}} ** __
```

```suggestion
    {{{emoji}}} {{{user}}} tarafından mesaj __ ** {{deleted}}/{{total}}} ** __
```
Looks like extra curly brackets are getting added for tr, cs, el, fi, ja, ko, lt, nl, no, th and zh translations. Could you please check? 🤔